sftp/1.3/modules/ROOT/edits/sftp-connection.adoc

```diff
@@ -0,0 +1,87 @@
+= Connect to an SFTP Server Example - Mule 4
+
+The following examples shows how to configure Anypoint Connector for SFTP (SFTP Connector) global configuration element to:
+
+* <<connect-sftp-server,Connect to an SFTP Server>>
+* <<connect-sftp-server-proxy,Connect to an SFTP Server Through a Proxy>>
+
+[[connect-sftp-server]]
+
+== Connect to an SFTP Server
+
+To configure the global element to connect to an SFTP Server, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. Select the operation in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (*+*) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, specify the connection information for the connector:
```

Comment:
period, not colon
If we are pointing to substeps, then they must be formatted as such